### PR TITLE
Added character limit to UILabel extension

### PR DESCRIPTION
**Pull request by imjog**
## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [x] New extensions are written in Swift 4.
- [x] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+.
- [ ] I have added tests for new extensions, and they passed.
- [x] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [x] All extensions are declared as **public**.
- [ ] I have added a changelog entry describing my changes.

### CODE REVIEW

File: Sources/Extensions/UIKit/UILabelExtensions.swift [hunk 1]
```diff
@@ -29,6 +29,27 @@ public extension UILabel {
 		label.sizeToFit()
 		return label.frame.height
 	}
+    
+    /// SwifterSwift: Add limit to number of character in UILabel text
+    /// replace characters after limit to '...'
+    public func addLimit(charracters: Int) {
```
Comment (SD10) [inline Sources/Extensions/UIKit/UILabelExtensions.swift]: Doesn't `NSLineBreakMode` handle this?

**Comment by SD10** [comment]
I'm not sure about this extension but if the community agrees it is a good idea then we definitely need to add tests.

**Comment by imjog** [comment]
well, i am closing the PR. i thought it would be helpful. thanks for reviews :)

**Comment by SD10** [comment]
@imjog We appreciate you taking the time to put together this PR 👍

**Pull request closed without merging**